policyengine_us/parameters/gov/states/me/tax/income/credits/fairness/property_tax/benefit_base/joint_or_head_of_household_multiple_children.yaml

```diff
@@ -4,6 +4,7 @@ values:
   2021-01-01: 3_400
   2022-01-01: 3_450
   2023-01-01: 3_700
+  2024-01-01: 3_900
```

Comment:
can you try to find the statutory backing for the uprating? i dont see it here but seems to be in effect. or file an issue if you can't

Reply:
yes